package.json

```diff
@@ -41,12 +41,15 @@
     "react-redux": "^8.1.1",
     "react-router-dom": "^6.10.0",
     "rxjs": "^7.8.1",
-    "tailwindcss": "^3.1.8",
-    "typescript": "^5.1.3"
+    "tailwindcss": "^3.1.8"
   },
   "devDependencies": {
+    "@types/jest": "^29.5.12",
     "autoprefixer": "^10.4.15",
+    "jest": "^29.7.0",
     "postcss": "^8.4.28",
-    "tailwindcss": "^3.1.8"
+    "tailwindcss": "^3.1.8",
+    "ts-jest": "^29.1.2",
+    "typescript": "^5.4.2"
```

Comment:
not sure why the typescript was removed from the dependencies, and added only to dev dependencies?